lighthouse-extension/gulpfile.js

```diff
@@ -115,27 +114,18 @@ gulp.task('html', () => {
 });
 
 gulp.task('chromeManifest', () => {
-  const manifestOpts = {
-    buildnumber: false,
-    background: {
-      target: `scripts/${CONSUMERS.EXTENSION.dist}`,
-    },
-  };
   return gulp.src('app/manifest.json')
-  .pipe(chromeManifest(manifestOpts))
```

Comment:
this may have come from the original generator. Not entirely sure what it's supposed to do, maybe copy over deps? But since the only target we give is the script bundle, and because we copy that ourselves below after browserifying it, this also has no effect but copying over the manifest file (which we can do more simply)